docs/CHANGELOG.asciidoc

```diff
@@ -101,7 +101,20 @@ This section summarizes the changes in each release.
 [[release-notes-6.8.11]]
 == {kib} 6.8.11
 
-coming::[6.8.11]
+[float]
+[[security-update-6.8.11]]
+=== Security updates
+* In {kib} 6.8.11 and earlier, there is a denial of service (DoS) flaw in Timelion. Attackers can construct a URL that when viewed by a {kib} user, 
+the {kib} process consumes large amounts of CPU and becomes unresponsive,
+https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2020-7016[CVE-2020-7016].
++
+You must upgrade to 6.8.11. If you are unable to upgrade, set `timelion.enabled` to `false` in your kibana.yml file to disable Timelion.
```

Comment:
@lcawl Was this intended to be "_6.8.10_ and earlier"?

Reply:
Thanks @jportner, I've created https://github.com/elastic/kibana/pull/75011